ci-operator/step-registry/camel-quarkus/execute-tests/camel-quarkus-execute-tests-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+shopt -s nullglob
+
+echo "Logging to oc:"
+sh ./oc_login.sh
+echo "Executing tests:"
+sh ./run.sh
+
+# Collect test reports into $ARTIFACT_DIR
+PROJECTS=$PROJECTS
```

Comment:
what is the need of assigning PROJECTS value to same variable here? It should work without reassigning it.

Reply:
Without the assignment shellcheck fails.

Reply:
I've tried